worlds/landstalker/__init__.py

```diff
@@ -220,14 +221,19 @@ def get_starting_health(self):
             return 4
 
     @classmethod
-    def stage_post_fill(cls, multiworld):
+    def stage_post_fill(cls, multiworld: MultiWorld):
         # Cache spheres for hint calculation after fill completes.
-        cls.cached_spheres = list(multiworld.get_spheres())
+        cached_spheres = list(multiworld.get_spheres())
+        for world in multiworld.worlds.values():
```

Comment:
Switched to using `get_game_worlds`. (I was using .values instead of .items because `player` was unneeded.)

How would I include or exclude itemlink worlds specially? The current usage is only for worlds of the same class, i.e. the same game. They all would use the same object, so it shouldn't matter if I include when I don't need to. And it looks like worlds created by `add_group` are added to `worlds` so should be covered by both `get_game_worlds` as well as what I was doing before.

Reply:
Ah, as in skips when `player` is a group id and not a player id. Looks like this is only used by `fill_slot_data` which is only called for player id worlds. Great.